js/catchTheFlow.js

```diff
@@ -115,9 +115,9 @@ function startingGame()
 function update()
 {
 	updateTimer();
-        
-        game.input.enabled=true; 
-   
+
```

Comment:
Only those changes you made should be reflecting here. That would be easy for me and other contributors to check and verify your changes.